examples/java/build.gradle

```diff
@@ -67,7 +67,11 @@ dependencies {
   compile library.java.joda_time
   compile library.java.proto_google_cloud_datastore_v1
   compile library.java.slf4j_api
-  compile library.java.slf4j_jdk14
+  compile "com.google.api.grpc:proto-google-cloud-language-v1:1.81.4"
+  compile "com.google.code.findbugs:jsr305:3.0.2"
+  compile "com.google.oauth-client:google-oauth-client:1.31.0"
+  permitUnusedDeclared "com.google.auto.value:auto-value-annotations:1.7.2"
+  permitUnusedDeclared "org.checkerframework:checker-qual:3.7.0"
```

Comment:
Adding dependencies to permitUnusedDeclared or permitTestUnusedDeclared causes the dependency analyzer to ignore cases where the dependencies are declared but not used.

Reply:
We prefer to use references to `library.auto_value_annotations`, etc. You may have to add some.

Reply:
@kennknowles  modified almost all the newly added dependencies to the library.auto_value_annotations now.

Reply:
... but not this one?

I wonder if you forgot to push the new commit?